requirements-dev.txt

```diff
@@ -58,3 +58,6 @@ python-jenkins==1.7.0
 testfixtures==6.17.1
 alchemy-mock==0.4.3
 factory-boy==3.2.0
+
+# pywatchman gives us watchman-wait which we use for watching translations.
+pywatchman==1.4.1
```

Comment:
when I run `docker compose exec web /bin/bash -c "source venv/bin/activate && yarn translations"`, I get the following error
```
Starting translations watch server...
Scanning files...
Found 173 files that import I18N
Collecting all translations...
Validating translations...
Updating i18n.js files...
10 files have updated translations
Resolving i18n imports...
Successfully updated all translation files
/bin/sh: 1: watchman-wait: not found
Done in 14.05s.
```

Reply:
You may just need to do a `docker compose build` - that will probably fix it.

There are some wierdsies around watchman (like it times out the 1st time you run it) - that for the sake of expediency I'm just going to park as is in this PR.

I'd rather just pull it out entirely (given that the code base didn't have this in!) - but seems that translations are pretty key?